test/language/generators/generator.declareMethod.shadow-parameter-catch.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+ GeneratorMethod early SyntaxError when lexical declaration
+ inside generator shadows parameter name 
+es6id: 14.4.1
+author: Sam Mikes
+description: GeneratorMethod error with lexical shadowing
+negative: SyntaxError
+---*/
+
+var obj = {
+    *foo(a) {
+        try {
+        }
+        catch (e) {
+            let a = 3;
```

Comment:
Should this really be an error? I think you can shadow formals, but you can't redeclare them (eg. let a = 3 in the generator body would be an error).